pkg/dynamicinstrumentation/diconfig/dwarf.go

```diff
@@ -16,10 +16,9 @@ import (
 	"reflect"
 	"slices"
 
-	"github.com/go-delve/delve/pkg/dwarf/godwarf"
-
 	"github.com/DataDog/datadog-agent/pkg/dynamicinstrumentation/ditypes"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/go-delve/delve/pkg/dwarf/godwarf"
```

Comment:
nit: external imports should be in a separate import group (separated by empty line). It was correct before, so it is likely you need to update your dev environment/IDE to sort imports correctly.